src/server/middlewares/handleRender.js

```diff
@@ -71,7 +71,7 @@ const Html = ({ content, state }) => (
     <body>
       <div id='root' dangerouslySetInnerHTML={{ __html: content }} />
       <script dangerouslySetInnerHTML={{
-        __html: `window.__APOLLO_STATE__=${JSON.stringify(state)};`
+        __html: `window.__APOLLO_STATE__=${JSON.stringify(state)};window.__CONFIG__=${JSON.stringify(config.get('server'))}`
```

Comment:
**Update**

I cannot remove `window._CONFIG_` because `SubscriptionClient` function still requires domain name in URL.